From 80946a41cd5283b50e844509f143825842524b8d Mon Sep 17 00:00:00 2001
From: Rong Dian <b38775@freescale.com>
Date: Thu, 24 May 2012 16:05:31 +0800
Subject: ENGR00210694 MX6 SABRESD:Enable IRQ for max11801

If miss to configure IRQ for max11801, the max11801 driver fails to
probe and returns error,sabresd battery driver also fails to sample
voltage by max11801 ADC.

Signed-off-by: Rong Dian <b38775@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index e749bc4..1af9aa6 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -787,6 +787,7 @@ static struct i2c_board_info mxc_i2c1_board_info[] __initdata = {
 	{
 		I2C_BOARD_INFO("max11801", 0x48),
 		.platform_data = (void *)&max11801_mode,
+		.irq = gpio_to_irq(SABRESD_TS_INT),
 	},
 };
 
-- 
1.7.9.5

